java/pom.xml

```diff
@@ -28,6 +29,15 @@
             </plugin>
         </plugins>
     </build>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>${logback.version}</version>
```

Comment:
Why not have it in one place like the other deps?

Reply:
See the second use of dependency management in https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management

In short, it ensures that any transitive logback dependency has the version number specified in the pom instead of using maven's default "distance from root" conflict resolution algorithm.